test/integration/webhooks/trainjob_test.go

```diff
@@ -161,6 +161,22 @@ var _ = ginkgo.Describe("TrainJob Webhook", ginkgo.Ordered, func() {
 						Obj()
 				},
 				testingutil.BeForbiddenError()),
+			ginkgo.Entry("Should succeed in creating trainJob with dataset config when referencing a ClusterTrainingRuntime with an initializer",
```

Comment:
We are no longer have Dataset config in our APIs.

```suggestion
			ginkgo.Entry("Should succeed in creating trainJob with dataset initializer when referencing a ClusterTrainingRuntime with an initializer",
```